src/main/java/org/cloudfoundry/promregator/lite/discovery/SimpleAppInstanceScanner.kt

```diff
@@ -0,0 +1,109 @@
+package org.cloudfoundry.promregator.lite.discovery
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import java.util.regex.Pattern
+
+private val log = mu.KotlinLogging.logger {  }
+
+@Component
+class SimpleAppInstanceScanner(
```

Comment:
With the coroutines refactor the instance scanner no longer needs to talk to the CFAccessor, this is now a simple class that calculates the instance access URLs. I kept that logic largely the same for now.